recipes/ninja/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "NINJA_nj" %}
+{% set version = "0.97" %}
+{% set sha256 = "b9b948c698efc3838e63817f732ead35c08debe1c0ae36b5c74df7d26ca4c4b6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/TravisWheelerLab/NINJA/archive/{{ version }}-cluster_only.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patch
```

Comment:
The file "patch" appears to be missing (and you could use a more specific name unless the best description is "general build fixes").